From 924dabd629da7b61c10da386f3a21b8eea1ef89c Mon Sep 17 00:00:00 2001
From: Marek 'marx' Grac <mgrac@redhat.com>
Date: Wed, 18 Feb 2015 14:22:37 +0100
Subject: [PATCH 05/16] Revert "COMPATIBILITY BREAK: remove -q / quiet"

This reverts commit 9049b8e08dbf93dbaa7507dedb7d36067852c011.

    Conflicts:
        fence/agents/lib/fencing.py.py
---
 fence/agents/lib/fencing.py.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/fence/agents/lib/fencing.py.py b/fence/agents/lib/fencing.py.py
index d4bb866..62f0df2 100644
--- a/fence/agents/lib/fencing.py.py
+++ b/fence/agents/lib/fencing.py.py
@@ -47,6 +47,11 @@ all_opt = {
 		"required" : "0",
 		"shortdesc" : "Display version information and exit",
 		"order" : 53},
+	"quiet"   : {
+		"getopt" : "q",
+		"longopt" : "quiet",
+		"help" : "",
+		"order" : 50},
 	"verbose" : {
 		"getopt" : "v",
 		"longopt" : "verbose",
@@ -434,7 +439,7 @@ all_opt = {
 
 # options which are added automatically if 'key' is encountered ("default" is always added)
 DEPENDENCY_OPT = {
-		"default" : ["help", "debug", "verbose", "version", "action", "agent", \
+		"default" : ["help", "debug", "verbose", "quiet", "version", "action", "agent", \
 			"power_timeout", "shell_timeout", "login_timeout", "power_wait", "retry_on", "delay"],
 		"passwd" : ["passwd_script"],
 		"sudo" : ["sudo_path"],
-- 
1.9.3

